server/pbenchinacan/load_keycloak.sh

```diff
@@ -18,12 +18,44 @@
 # env variable.
 
 KEYCLOAK_HOST_PORT=${KEYCLOAK_HOST_PORT:-"http://localhost:8090"}
+KEYCLOAK_HEALTH_URI="${KEYCLOAK_HOST_PORT}/health"
```

Comment:
@npalaska, what are our expectations for this PR vis a vis PR #3208?

That is, this file seems to appear in both PRs with similar changes in each.  I think it would be better if you based one of the two PR branches on the other one (and maybe held the second in draft mode until we can land the first).

Alternatively, perhaps you should just merge the two branches into a single PR?

Reply:
I believe the first three commits here *are* #3208 (although that PR now has 4 commits).

I think that ideally I'd like to see #3208 refactor the keycloak configuration into a script acting against an already-running Keycloak broker. However, to integrate that into our CI we need to pull and stand up a Keycloak and *then* modify it, which (so we think) may be complicated to manage with the pod YML.

So I believe the logic was to go as far as he could with #3208 and then do the rest along with the cutover to separate `podman` commands separately.

Merging into one would be fine with me. Or, to keep them small and focused, just do the "YML to command" conversion first, then rebase #3208 on top with the addition of running the setup script after launching the broker...

Reply:
I think we can start by (e.g., in #3208) creating the script which will configure a running instance of a Keycloak server and merge that in, even though nothing automated uses it (we can use it on Staging).  Then we can blow up the pod (e.g., in this PR).  And, _then_ either in this PR or a follow-on, we can modify the CI to invoke the script created in #3208.  (Or, we can do it all in one PR, but, since #3208 is really close to being "done", I'd like to see that merged, and then we can merge this one, and then we can merge the next, rather than waiting for all the ducks to be in a row.)

Reply:
Yeah as I said this PR is currently rebased on top of PR https://github.com/distributed-system-analysis/pbench/pull/3208, therefore, you see commits from that PR here and it is dependent on getting #3208 merge first. I opened this PR to get some feedback. I should have just kept it draft and asked for the review explicitly.

Reply:
> as I said this PR is currently rebased on top of PR https://github.com/distributed-system-analysis/pbench/pull/3208

Ah, so you did.  I missed that -- my bad.